inc/lift/RequestHandle.h

```diff
@@ -254,8 +242,8 @@ class RequestHandle
 
     void* m_user_data;                          ///< The user data.
 
-    size_t m_max_bytes;                         ///< Maximum number of bytes to be written.
-    size_t m_bytes_left_to_write;                ///< Difference between maximum bytes to write and amount written
+    ssize_t m_max_download_bytes;                         ///< Maximum number of bytes to be written.
+    ssize_t m_bytes_left_to_write;               ///< Difference between maximum bytes to write and amount written
     bool m_wrote_max_bytes;                     ///< True if max number of bytes have been written
```

Comment:
do we really need a bool value for this? its can be calculated by subtracting two fields correct?  I _think_ just having m_max_download_bytes and m_total_bytes_received should be enough to implement this feature with maybe some helper private member functions

Reply:
Good point! Cut down to only m_max_download_bytes and m_bytes_written and add getBytesLeftToWrite.